cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/StructuredTableAdmin.java

```diff
@@ -26,6 +30,24 @@
    * Create a StructuredTable using the {@link StructuredTableSpecification}.
    *
    * @param spec table specification
+   * @throws IOException if there is an error creating the table
+   */
+  void create(StructuredTableSpecification spec) throws IOException;
```

Comment:
What if the table already exist? Will it throw, update or no-op?

Reply:
Added `StructuredTableAlreadyExistException` in case the table already exist